dashboard/test/lib/pd/payment/payment_factory_test.rb

```diff
@@ -26,8 +26,7 @@ class PaymentFactoryTest < ActiveSupport::TestCase
       workshop_ecs = create :workshop, :ended, on_map: false, funded: true,
         course: Pd::Workshop::COURSE_ECS, subject: Pd::Workshop::SUBJECT_ECS_PHASE_2
 
-      workshop_csp = create :workshop, :ended, on_map: true, funded: true,
-        course: Pd::Workshop::COURSE_CSP, subject: Pd::Workshop::SUBJECT_CSP_WORKSHOP_1
+      workshop_csp = create :csp_academic_year_workshop, :ended, on_map: true, funded: true
```

Comment:
these are funded by default, but maybe worth keeping the `funded: true` to be explicit since this is a payment-related test?

Reply:
Yeah, in a few cases I left redundant properties in because it seemed like it was specifically relevant to the test.  This was one of those.